internal/codespaces/api/api_test.go

```diff
@@ -140,7 +140,7 @@ func TestListCodespaces_limited(t *testing.T) {
 		client:    &http.Client{},
 	}
 	ctx := context.TODO()
-	codespaces, err := api.ListCodespaces(ctx, 200, "", "")
+	codespaces, err := api.ListCodespaces(ctx, ListCodespacesOptions{Limit: 200, OrgName: "", UserName: ""})
```

Comment:
Can we just remove the defaults now?

Reply:
like `OrgName` and `UserName` yeah that makes sense